scripts/invoke.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# When running the zat container, mount the current directory to /app
+# so that zat has access to it.
+docker run -it --rm -v $(pwd):/usr/src/app zat $@
```

Comment:
hmm... I thought you already copy the src folder as part of the docker build. Why do you need to mount it again?

In addition, if you need to mount it for running unit tests changes, do you need to follow the Zendesk environment variable to use that to point the zat repository under your code path?

Reply:
I think this is intended for use in app directories, hence it's mounting app source codes rather than ZAT code.

Though by the way I believe `$(pwd):/usr/src/app` and `$@` both need to be quoted:
- https://github.com/koalaman/shellcheck/wiki/SC2046
- https://github.com/koalaman/shellcheck/wiki/SC2068

Reply:
oh, so $(pwd) indicating the application code base folder instead of zat project. I see now

Reply:
ah, I see the issue. I need to alias this command as eg `zatDev`, or write a shortcut to mount it within ~/.zshrc or somesuch

basically the usecase is

```
cd mycoolapp
zatDev apps server
```

so we'd run the container and mount the mycoolapp directory within it, so that `zatDev apps server` notices when things change and updates the served app accordingly

This ask actually came from `zat themes preview`, we want to mount a theme and live poll for updates to the directory on the machine which we mount as a volume relative to the running container.